### ISSUE TITLE
Phantomjs rendering broken in development mode

### ISSUE THREAD

**Issue by marefr** [grafana/grafana#11883]
Please include this information:
### What Grafana version are you using?
master
### What did you do?
`npm run watch`
Then tried to generate an image from panel using panel -> share -> link -> Direct link rendered image
### What was the expected result?
An image is rendered correctly
### What happened instead?
Render of image is timing out

Also did a `npm run build` to validate if the same problem exist for production build, but there it's working as expected.

Did a git bisect and identified the following commit (https://github.com/grafana/grafana/commit/cc5d7002b0e5fa12790a366696bc369ec95be379) to have introduced this problem.

Confirmed that removing the [following lines](https://github.com/grafana/grafana/blob/master/scripts/webpack/webpack.dev.js#L70-L78) makes the phantomjs rendering work.

So basically it's a typescript/babel issue since typescript are transpiling to es5 and babel which runs after expects es6.

Tried suggestion [here](https://github.com/gaearon/react-hot-loader#add-babel-after-typescript) without any good result.

There's also another suggestion [here](https://github.com/gaearon/react-hot-loader#add-babel-before-typescript) which I haven't tried.
